dcpy/lifecycle/package/generate_metadata_assets.py

```diff
@@ -8,16 +8,21 @@
 DEFAULT_DATA_DICTIONARY_TEMPLATE_PATH = (
     RESOURCES_PATH / "data_dictionary_template.jinja"
 )
+DEFAULT_DATA_DICTIONARY_STYLESHEET_PATH = RESOURCES_PATH / "data_dictionary.css"
```

Comment:
noting that I'm not sure if these `DEFAULT_*` variables are the best approach, just wanted to minimize changes in this PR